src/sagemaker_training/smdataparallel.py

```diff
@@ -284,18 +284,20 @@ def run(self, wait=True, capture_error=False):
         cmd = self._create_command()
         cmd.extend(super(SMDataParallelRunner, self)._create_command())
         logging_config.log_script_invocation(cmd, self._env_vars)
+
+        exception_classes_all = exception_classes + process.get_debugger_exception_classes()
```

Comment:
can we change this to `exception_classes += process.get_debugger_exception_classes()` in order for minimal code changes?